foregroundviews/src/main/java/com/commit451/foregroundviews/ForegroundLinearLayout.java

```diff
@@ -19,17 +19,16 @@
 
 package com.commit451.foregroundviews;
 
-import android.annotation.TargetApi;
 import android.content.Context;
 import android.graphics.Canvas;
 import android.graphics.drawable.Drawable;
 import android.os.Build;
+import android.support.v7.widget.LinearLayoutCompat;
 import android.support.annotation.NonNull;
 import android.util.AttributeSet;
-import android.widget.LinearLayout;
 
 
-public class ForegroundLinearLayout extends LinearLayout {
+public class ForegroundLinearLayout extends LinearLayoutCompat {
```

Comment:
Yes, you are right, I think it's should not be here. Just I used LinearLayoutCompat in current project and it was need.